src/lotus_json/mod.rs

```diff
@@ -355,10 +355,11 @@ impl<T> Stringify<T> {
     }
 }
 
+#[macro_export]
```

Comment:
I don't think you should do this, it exposes the macro as part of our public api:
https://doc.rust-lang.org/beta/reference/macros-by-example.html#scoping-exporting-and-importing

add `pub(crate) use lotus_json_with_self` and use it like a normal item instead